frontend/components/card/card.css

```
@@ -15,6 +15,21 @@
   -ms-user-select: none;
   user-select: none;
 }
+.modify {
+  cursor: pointer;
+  padding: 16px;
+  width: 308px;
+  min-height: 128px;
+
+  background: var(--white);
+  border: 1px solid var(--purple-100);
+  border-radius: 6px;
+
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
```

스타일 굿입니다!

.card와 중복되는 스타일 값들은 삭제해도 될 거 같아요!

```suggestion
.modify {
  border: 1px solid var(--purple-100);
}
```

---

넵 제거 했습니다!